### ISSUE TITLE
About the backbone checkpoint

### ISSUE THREAD

**Issue by ttt-y** [donydchen/mvsplat#97]
Hello, Thanks for your excellent work!
I've tested the evaluation part of the work. Now I want to train the model to  observe the initial outputs and how the loss changes. However, I failed to download the backbone pretrained weight using the command:
 `wget 'https://s3.eu-central-1.amazonaws.com/avg-projects/unimatch/pretrained/gmdepth-scale1-resumeflowthings-scannet-5d9d7964.pth' -P checkpoints`
 I wasn't able to access the website, and the download failed. Could this be due to my internet connection, or is there an alternative way to obtain the weights?
Thanks a lot!!!

**Comment by ttt-y** [comment]
sorry, I’d solved the problem.
